src/Opdex.Platform.Application/EntryHandlers/Transactions/TransactionLogs/MarketDeployers/ProcessCreateMarketLogCommandHandler.cs

```diff
@@ -39,6 +39,8 @@ public async Task<bool> Handle(ProcessCreateMarketLogCommand request, Cancellati
             if (market != null) return true;
 
             // Get potential market staking token
+            // Todo: We _should_ add this token if it doesn't exist however, the contract will return Address.Zero which looks like a real address
+            // -- Would need to attempt to get the token summary to see if it exists, or add Address.Zero per market to check.
             var stakingToken = await _mediator.Send(new RetrieveTokenByAddressQuery(request.Log.StakingToken, findOrThrow: false));
```

Comment:
@drmathias I suspect this is what you might have run into when creating the market. We'll come back to it.